widget/slider.go

```diff
@@ -121,11 +127,21 @@ func (s *Slider) positionChanged(lastValue, currentValue float64) {
 
 	s.Refresh()
 
+	s.pendingChange = true
 	if s.OnChanged != nil {
 		s.OnChanged(s.Value)
 	}
 }
 
+func (s *Slider) checkInvokeOnChangeEnded() {
+	if s.pendingChange {
```

Comment:
We prefer early exit so main body is not indented. (I.e. invert logic and return instead)

Reply:
FYI @dweymouth: We prefer to have the reviewer mark their own comments as resolved :)